### PR TITLE
done

### PR DESCRIPTION
**Pull request by ArchanaManjuu**

### CODE REVIEW

File: githubpractice2/src/test/java/dummy/NewTest.java [hunk 1]
```diff
@@ -5,7 +5,11 @@ public static void main(String[] args) {
 		System.out.println("dum");
 		System.out.println("dum");
 		System.out.println("dum2");
-		System.out.println("dum2");
+	//	System.out.println("dum2");
+
```
Comment (Archanathoughts) [inline githubpractice2/src/test/java/dummy/NewTest.java:9]: uncomment the last statement

**Review by Archanathoughts — CHANGES_REQUESTED** [review]
plz do the changes

File: githubpractice2/src/test/java/dummy/NewTest.java (hunk 1 above)
Comment (ArchanaManjuu) [reply githubpractice2/src/test/java/dummy/NewTest.java:9]: done the changes

**Review by Archanathoughts — COMMENTED** [review]
approved

File: githubpractice2/src/test/java/dummy/NewTest.java (hunk 1 above)
Comment (Archanathoughts) [reply githubpractice2/src/test/java/dummy/NewTest.java:9]: merged

**Pull request merged**